addons/account/models/company.py

```diff
@@ -190,6 +190,11 @@ class ResCompany(models.Model):
             ('out_and_in_invoices', 'Customer Invoices and Vendor Bills')],
         string="Quick encoding")
 
+    show_invoice_tax = fields.Boolean(
+        compute='_compute_show_invoice_tax',
+        store=True,
+        help="Used by localization packages, False if the package wants to hide the taxes column",)
```

Comment:
just received an email and check the pr by curiosity, a small nitpicking could be to put the parenthesis on the next line to avoid breaking the github history if someone work on the field later on.

Also maybe an export of the pot would be nice for that (no need for po file i think), what do you think @hupo-odoo ?

Reply:
I agree with the parenthesis :+1: 

The task is in master and the field is added in a non-localization module so why should we add the pot file ?